Docs for v0.5.0 (#24)
stackrlabs/docs: docs/pages/build/framework/block.mdx

- `operatorSignature`: The signature of the operator on the block.
- `builderSignature`: The signature of the builder on the block.
- `executionStatus`: The status of the block execution. It is an encoded value that represents which action succeeded and which failed.

> - `batchInfo` : The information about the batch to which the block belongs.

Reviewer: ```suggestion
- `batchInfo`: Metadata of the batch that includes this block.
```

Reviewer: Also, add that it's not filled by default and is fetched later from Vulcan
